src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1246,6 +1308,16 @@ impl<'a> QemuCmdLine<'a> {
         self.devices.push(Box::new(rtc));
     }
 
+    fn add_rng_object(&mut self) {
+        let rng_object = ObjectRngRandom::new();
+        self.devices.push(Box::new(rng_object));
+    }
+
+    fn add_rng_device(&mut self) {
+        let rng_device = DeviceRng::new();
+        self.devices.push(Box::new(rng_device));
+    }
```

Comment:
This should be a single `add_rng()` function that instantiates and stores all required objects (see other `QemuCmdLine::add_*()` functions for an example).